src/server/replica.cc

```diff
@@ -1229,7 +1229,7 @@ bool Replica::TransactionReader::ReturnEntryOOO(const TransactionData& tx_data,
 auto Replica::TransactionReader::NextTxData(JournalReader* reader, Context* cntx)
     -> optional<TransactionData> {
   io::Result<journal::ParsedEntry> res;
-  TransactionData next_tx;
+  TransactionData next_tx{};
```

Comment:
Why its important?

Reply:
uninitialized warnings

Reply:
Maybe, instead add c'tor for TransactionData, or  initialize the fields inside the struct than relying on this convention (that must be repeated every time you declare a var). I assume here that the c'tor or field initialization solves the warning too. 

by the way, I did not know there is a difference between "V var" and "V var{}" for non-integral types.